pkg/activator/dedupe_test.go

```diff
@@ -16,12 +16,13 @@ limitations under the License.
 package activator
 
 import (
-	"fmt"
 	"net/http"
 	"reflect"
 	"sync"
 	"testing"
 	"time"
+
+	"github.com/pkg/errors"
```

Comment:
In this case, this import can just be "errors". You're not using `Wrap`, `Wrapf` and friends here I think?

Reply:
Yeah, I trusted the automation and didn't even check those. Why goimports would prefer that one is a puzzle to me. :astonished: